### PR TITLE
2022-07-10 AKS Release Notes

### PR DESCRIPTION
**Pull request by CocoWang-wql**
2022-07-10 AKS Release Notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+
+### Release notes
+* Features
+  * [Microsoft Defender cloud-native security agent for AKS clusters ](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-containers-enable?tabs=aks-deploy-portal%2Ck8s-deploy-asc%2Ck8s-verify-asc%2Ck8s-remove-arc%2Caks-removeprofile-api&pivots=defender-for-container-aks) is now generally available. 
+  * [AKS API Server VNet Integration](https://docs.microsoft.com/azure/aks/api-server-vnet-integration) is now generally available. 
+
+* Component Updates
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2022.07.04](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2022.07.04.txt).
+  * AKS Windows 2019 image has been updated to [17763.3165.220713](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSWindows/2019/17763.3165.220713.txt).
+  * AKS Windows 2022 image has been added with version [20348.825.220713](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSWindows/2022-containerd/20348.825.220713.txt).
```
Comment (shashankbarsin) [inline CHANGELOG.md:29]: these two files are for 07-13, so they wouldn't have made it 07-10 release. we need to exclude these two. were there any images between 07-03 and 07-10 that were released?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled.  This will allow users to enable/disable node restriction.
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Kubernetes patch versions 1.21.14, 1.22.11, and 1.23.8 are now available; Kubernetes patch versions 1.21.7, 1.22.4, and 1.23.3 are deprecated and removed. Learn more about Kubernetes version support policy followed by AKS [here](https://docs.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy).
```
Comment (shashankbarsin) [inline CHANGELOG.md:18]: we had this announcement of the new & deprecated versions in the previous release, we can remove this line

File: CHANGELOG.md (hunk 1 above)
Comment (CocoWang-wql) [reply CHANGELOG.md:29]: Nope, confirmed with Abel, these two are the next version after 2022-06-24. Let me remove them then.

**Review by shashankbarsin — APPROVED** [review]

**Review by justindavies — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is done for all regions. 
+
+### Release notes
+* Features
+  * [Microsoft Defender cloud-native security agent for AKS clusters ](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-containers-enable?tabs=aks-deploy-portal%2Ck8s-deploy-asc%2Ck8s-verify-asc%2Ck8s-remove-arc%2Caks-removeprofile-api&pivots=defender-for-container-aks) is now generally available. 
+  * [AKS API Server VNet Integration](https://docs.microsoft.com/azure/aks/api-server-vnet-integration) is now generally available.
```
Comment (palma21) [inline CHANGELOG.md:23]: @miwithro @phealy I'm guessing we're not GAing this yet?

@justindavies @shashankbarsin

**Pull request merged**